### ISSUE TITLE
Would this be compatible with LibreELEC?

### ISSUE THREAD

**Issue by nickrob84** [LS80/script.openelec.rpi.config#16]
Looking for something exactly like this but LibreELEC obviously!

**Comment by xbmcnut** [comment]
Yes, it works fine on LibreELEC. http://fusion.tvaddons.ag > repository.leopold-3.0.0.zip > Services > OpeneELEC RPi-Config

**Comment by nickrob84** [comment]
Thank you very much for your reply.

**Comment by GoodGuy98** [comment]
The LibreELEC repo posted above is missing. Is there a new repo for RPi-Config addon available?

**Comment by mildenhall** [comment]
I think the tvaddons repro was closed down for legal reasons.

Anyway get this addon direct from the source...

http://ls80.github.io/script.openelec.rpi.config/

**Comment by GoodGuy98** [comment]
I started to install using that web page, but the repo listed is missing.When I couldn't access that, I tried researching to find a newer repoand the tvaddons repo was missing also.

**Comment by mildenhall** [comment]
Use google!

A simple google of "leopold repository" gives you https://github.com/LS80/repository.leopold

**Comment by GoodGuy98** [comment]
Thank you.This stuff is new to me, especially GitHub.
